src/server/scripts/Outland/HellfireCitadel/MagtheridonsLair/boss_magtheridon.cpp

```diff
@@ -234,6 +248,13 @@ struct boss_magtheridon : public BossAI
         BossAI::JustEngagedWith(who);
         Talk(SAY_EMOTE_BEGIN);
 
+        me->GetCreaturesWithEntryInRange(_channelers, 200.0f, NPC_HELLFIRE_CHANNELER);
```

Comment:
Use a local variable for this considering its not used anywhere else outside this scope

Reply:
Additionally, you may also use
```cpp
instance->DoForAllMinions();
```